### ISSUE TITLE
Response buttons not working since reinstall

### ISSUE THREAD

**Issue by touchnova** [Exslims/MercuryTrade#256]
~~I recently had to send my gaming laptop in to have the mobo replaced.  When I got it back I decided I wanted to do a fresh install of Mercury Trade to ensure latest versioning with the new Bestiary league.  I manually deleted the Mercury Trade folder out of "appdata/local/" and reinstalled.  However, when I set up my response buttons and did the test for incoming trades, my response buttons are not popping up chat msgs like they used to.  I click a button ('1m' for example = 'one minute plz') and nothing happens.  As well, when I have a friend initiate a trade for testing, I only get the standard PM chat sound in PoE, not the Mercury Trade notification sound and the trade pop-up doesn't appear.  Is there a better way for me to uninstall and reinstall fresh, or any other insight?  Thanks much in advance.~~

~~(notes - latest Java version is installed, and I used the .exe to install as the standalone .jar version previously gave me minor functionality issues)~~

**Comment by touchnova** [comment]
~~I cleared "appdata/local/MercuryTrade" again and tried the .jar version rather than the .exe.  Now the pop-up windows are working, but the button/chat functionality is still broken. Per Exslims suggested fix on Discord, I ensured that _"Put tooltip results on clipboard" in the iteminfo tab in poe trade macro_ was unchecked (in my case it was unchecked by default).  This fix did not resolve the issue for me.~~
